### PR TITLE
Handle no SSL termination in the api-gateway

### PR DESCRIPTION
**Pull request by nucatus**
In case no SSL termination is desired in the api-gateway, that should be
possible and the traefik configuration should be consistent with the
supplied configuration parameter `api_gateway.env.SSL`.

Signed-off-by: Alexandru Ionita <alexandru.ionita@gmail.com>

### CODE REVIEW

File: mender/templates/api-gateway-configmap.yaml [hunk 1]
```diff
@@ -1,3 +1,5 @@
+{{ $entrypoint := ternary "https" "http" (.Values.api_gateway.env.SSL) }}
```
Comment (ffoysal) [inline mender/templates/api-gateway-configmap.yaml:1]:
```suggestion
{{ $scheme := ternary "https" "http" (.Values.api_gateway.env.SSL) }}
```
with this single variable might refer to everywhere.

File: mender/templates/api-gateway-configmap.yaml [hunk 2]
```diff
@@ -417,10 +420,11 @@ data:
             regex: "^/api/management/v1/inventory/(.*)"
             replacement: "/api/0.1.0/$1"
 
+{{- if .Values.api_gateway.env.SSL }}
```
Comment (ffoysal) [inline mender/templates/api-gateway-configmap.yaml:423]:
```suggestion
{{- if $isTls }}
```

Comment (nucatus) [reply mender/templates/api-gateway-configmap.yaml]: Comments accepted and addressed in commit 53d7e06.

File: mender/templates/api-gateway-configmap.yaml (hunk 1 above)
Comment (nucatus) [reply mender/templates/api-gateway-configmap.yaml]: Comments accepted and addressed in commit 53d7e06.

**Review by ffoysal — APPROVED** [review]

**Review by tranchitella — APPROVED** [review]
Looks good, thank you for your contribution

**Pull request merged**